apiserver/src/helpers/Token/UserInScope.ts

```diff
@@ -0,0 +1,17 @@
+import { NextFunction, Request, Response } from "express";
+import NotANumberError from "../Errors/NotANumberError";
+
+export function addUserIdToScope(req: Request, userId: number): void {
+	req.body = {
+		userId: userId.toString(),
+		...req.body,
+	};
+}
+
+export function getUserIdFromScope(req: Request, _res: Response, _next: NextFunction): number {
+	const userId = parseInt(req.body.userId);
```

Comment:
Not sure the userId is an number... (If we're talking about the firebase uid ?)
You probably tested it though...

Reply:
I think the user Id returned by Auth.getOne is a number ... Will be reviewed and clean with Typescript enforcement during #4